trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0...10]
+  end
+end
```

Comment:
- [ ] Good to implement the method `correct_name` but make sure that it output of `@nameable.correct_name` has a maximum of 10 characters so I think the index range should be `[0...9]` not `[0...10]` and only If the word is longer than 10 hence you also need to implement if statement here to handle that. like this :point_down: 
  
  ```ruby
   # The name has a maximum of 10 characters. If it's longer it should trim the word.
      def correct_name
        @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
      end
  ```

Reply:
This change has been done well